OpenCensusAgent/opencensus-service/Makefile

```diff
@@ -0,0 +1,128 @@
+# More exclusions can be added similar with: -not -path './vendor/*'
+ALL_SRC := $(shell find . -name '*.go' \
+                                -not -path './vendor/*' \
+                                -not -path './tools/*' \
+                                -type f | sort)
+
+# ALL_PKGS is used with 'go cover'
+ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
+
+GOTEST_OPT?=-v -race -timeout 30s
+GOTEST_OPT_WITH_COVERAGE = $(GOTEST_OPT) -coverprofile=coverage.txt -covermode=atomic
+GOTEST=go test
```

Comment:
why does it still say tests?

Reply:
I didn't modify the original make file. I can take out the parts related to tests here. The rest of the tests copied over from OCAgent should be gone.